package_testing/test.py

```diff
@@ -0,0 +1,26 @@
+from core.feluda import Feluda
```

Comment:
hi @plon-Susk7 wanted to ask a question, can we move the entire testing code out of feluda? and just run it in an isolated environment like google colab etc?

don't know what the ideal protocol is, hence asking
also what if we write new tests for such use cases? like using a config to init Feluda class and use the opeartor.

Reply:
hey @aatmanvaidya , here's an example that I ran in google colab https://colab.research.google.com/drive/1us_dvUZpC4hdVTXnnyGVkE7IvXNAKNfP?usp=sharing

Reply:
hi @plon-Susk7 I did take a look at the colab, the code there looks great!!
I wanted to ask if the test file in the code is necessary then? if yes, we should write like proper tests for it where we assert values etc -- this makes it more robust and easy to maintain

Reply:
I agree @aatmanvaidya. We can remove the the current file and instead add proper tests.

Reply:
great, so to just test the operator, we just need a unit test that extracts the embedding on an image
I think such a test for the operator might already exist, 

we should write a test that init's Feluda class and then uses the operator, extracts the embedding on an image (you can just use one of the links I had provided earlier), and asserts for the vector length

Reply:
@aatmanvaidya , I have added the test case.

Reply:
After initializing the virtual environment move to package_testing directory and run following command.
```cmd
uv run -m unittest test_image_vec_rep_resnet.py
```

Reply:
it looks great, one small thing, can you rename the folder from `package_testing` to `tests`. And the function you have written for `test_sample_image_from_disk()` is perfect, but I think we should not populate the codebase right now with more sample images, we already have enough of them, so you can skip the disk() function?

Reply:
done

Reply:
you don't have to delete that function - you can just add a `@skip` above the function
```py
from unittest.case import skip

@skip
test_sample_image_from_disk():
```
you can see this for example - [code file](https://github.com/tattle-made/feluda/blob/690d48a63a855cfa8675fd1099f57e8a6aacd8db/src/core/operators/test_vid_vec_rep_clip.py#L19-L21)

also, the `package_testing` folder is still there, can you also delete it?